test/Microsoft.ML.Functional.Tests/Microsoft.ML.Functional.Tests.csproj

```diff
@@ -0,0 +1,53 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AssemblyName>Microsoft.ML.Functional.Tests</AssemblyName>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
I don't think we need `unsafe` blocks (at least not right now). Let's leave this off for now.